share/misc/bsd-family-tree

```diff
@@ -910,8 +916,8 @@ Die freien BSD-Varianten von Unix. c't April 1997, page 368ff.
 BSD Release Announcements collection.
 URL: https://www.FreeBSD.org/releases/
 
-BSD Hypertext Man Pages
-URL: https://www.FreeBSD.org/cgi/man.cgi
+Manual pages for FreeBSD and ports
```

Comment:
This includes man pages for all the other BSDs, or did historically, so I think this is not a good change.

Reply:
Thanks. Revert both lines?

There's a redirect from the original URL (`www.`) to `https://man.freebsd.org/cgi/man.cgi` (`man.`)

Reply:
https://github.com/freebsd/freebsd-src/commit/e80697634978578bca84a6f28348db0fbc40118d affects this area

Reply:
f8952e4ee0889bcedd373e7b1b071ebfd701b8ae

Thanks, everyone.

Reply:
> This includes man pages for all the other BSDs, 

I see what you mean. <https://old.reddit.com/r/freebsd/comments/18ozm2o/-/> (23rd December) sang praises for (amongst other things) the presence of pages for OpenBSD.